master/master.cfg

```diff
@@ -66,15 +70,21 @@ from buildbot.status.results import SKIPPED
 
 def do_step_build(step, name):
     props = step.build.getProperties()
+
     override_name = 'override-' + name
     if props.hasProperty(override_name):
-        if props[override_name] == "yes":
+        m = re.match(r".*yes.*", props[override_name])
+        if m is not None:
             return True
         else:
             return False
 
-    if props.hasProperty(name) and props[name] == "yes":
-        return True
+    if props.hasProperty(name):
+        m = re.match(r".*yes.*", props[name])
```

Comment:
Was the regex match introduced because some properties are now surrounded by quotes?

Reply:
Yes, exactly.